src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -28,6 +28,12 @@
 import teammates.logic.core.FeedbackSessionsLogic;
 import teammates.logic.core.InstructorsLogic;
 import teammates.logic.core.StudentsLogic;
+import teammates.sqllogic.core.UsersLogic;
```

Comment:
There should already be an sqlEmailGenrator, changes here not required.

Reply:
reverted, didnt know there was such a class